main.js

```diff
@@ -1,5 +1,8 @@
 var table = document.querySelector("table");
 var gradeTable = new GradeTable(table);
 
-var app = new App(gradeTable);
+var head = document.querySelector("header");
```

Comment:
I would call this header because there is also an html head element and i get confused easily lol